nemo/collections/multimodal/data/energon/taskencoder.py

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import dataclasses
+from typing import Dict, List, Union
+
+import torch
+from megatron.energon import (
+    CaptioningSample,
+    DefaultTaskEncoder,
+    InterleavedSample,
+    SimilarityInterleavedSample,
+    VQASample,
+    batch_list,
+    batch_pad_stack,
+)
+
+from nemo.utils import logging
+
+from .config import ImageTextRawBatch, ImageTextSample
```

Comment:
NeMo avoids using relative importing, can we update to absolute path?

Reply:
Same below

Reply:
done